### PR TITLE
Add range syntax sugar for secret generation

### PR DESCRIPTION
**Pull request by njhale**
Add the ability to specify ranges "A-Z", "a-z", and "0-9" when providing
arguments for basic and token type secrets.

### CODE REVIEW

File: pkg/secrets/generate_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+func TestInflateRanges(t *testing.T) {
+	for _, tt := range []struct {
+		name string
+		want string
+	}{
+		{
+			name: "A-Z",
+			want: "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+		},
```
Comment (tylerslaton) [inline pkg/secrets/generate_test.go:17]: Can we add a test case for empty ranges being passed too? Mainly to make certain we don't have any chance of hitting an out of bounds panic.

**Review by tylerslaton — APPROVED** [review]

File: pkg/secrets/generate_test.go (hunk 1 above)
Comment (njhale) [reply pkg/secrets/generate_test.go:21]: done

**Review by StrongMonkey — APPROVED** [review]

**Review by tylerslaton — APPROVED** [review]
